addons/l10n_ec/models/account_move.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    l10n_ec_sri_payment_id = fields.Many2one(
+        comodel_name="l10n.ec.sri.payment",
+        string="Payment Method (SRI)",
+    )
+
+    l10n_ec_emission_type = fields.Selection(
+        string="Emission type",
+        selection=[
+            ("pre_printed", "Pre Printed"),
+            ("auto_printer", "Auto Printer"),
+            ("electronic", "Electronic"),
+        ],
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+        default="electronic",
+    )
+
+    l10n_ec_identification_type_id = fields.Many2one(
+        "l10n_ec.identification.type",
+        string="Ecuadorian Identification Type",
+        store=True,
+        compute="_compute_l10n_ec_identification_type",
+        compute_sudo=True,
+    )
+    l10n_ec_is_exportation = fields.Boolean(string="Is Exportation?")
```

Comment:
Can not this be derived from the country of the partner?

Reply:
computed behind the scened > partner from a foreign country > then is an exportation

Reply:
Reimplemetation, to python calc